shared/constants/types/chat2/index.js

```diff
@@ -24,6 +25,7 @@ export type _State = {
   messageMap: I.Map<Common.ConversationIDKey, I.Map<Message.Ordinal, Message.Message>>, // messages in a thread
   messageOrdinals: I.Map<Common.ConversationIDKey, I.SortedSet<Message.Ordinal>>, // ordered ordinals in a thread
   metaMap: I.Map<Common.ConversationIDKey, Meta.ConversationMeta>, // metadata about a thread
+  quotedMessage: ?HiddenString, // current message being quoted
```

Comment:
has to be based on conversationidkey. very dangerous for these to just fly around w/o any context

Reply:
@chrisnojima Tried that initially, but there's a "Reply Privately" option, and in that case we know which text you want to quote and who the author is, but we don't know a ConversationIDKey, and we're not going to know one until some actions finish running -- we'll be creating a new convo just for this, if you're replying privately to someone you don't have a convo with already.

So at the time we want to store away the quoted message, we won't know which convID to associate it with.  I guess we could do something where it gets stored away as a pending quote associated with the author it's from, and when a convo from that author is selected then it's removed from a pending set.  But it's so ugly.  Any better ideas?

It would help a lot if there was a way to:
```
yield Saga.put(
    Chat2Gen.createStartConversation({
      participants: [message.author],
    })
  }
```
.. but then access the started conversationID immediately after that yield.  Maybe that's what we should build?

Reply:
I've never been a huge fan of the UI sending message by name, but if you are having a hard time plumbing this stuff through, you can always do it by name here if it makes it easier. Let me know, and I can add it to the interface on the service.